roles/inject_openshift_kube_objects/tasks/main.yml

```diff
@@ -3,14 +3,23 @@
   shell: "mkdir -p {{ kube_objects_dir }}"
 
 - name: "Decode the symmetric key received from the user"
-  shell: "echo \"{{ symmetric_key }}\" | openssl enc -d -base64 > {{ kube_objects_dir }}/get_sym_key"
+  shell: "echo \"{{ symmetric_key }}\" | base64 --decode > {{ kube_objects_dir }}/get_sym_key"
   no_log: true
   
 - name: "Decrypt the symmetric key using RSA private key"
   shell: "openssl rsautl -decrypt -oaep -inkey {{ rsa_private_key }} -in {{ kube_objects_dir }}/get_sym_key -out {{ kube_objects_dir }}/secret.key"
 
+- name: "Verify the shared symmetric key"
+  shell: "stat -c %s {{ kube_objects_dir }}/secret.key"
+  register: symmetric_key_size
+
+- name: "Fail if the shared symmetric key is not 32 bytes"
+  fail:
+    msg: "Failed: The shared symmetric key should be of 32 bytes"
```

Comment:
can we add  "symmetric_key_size" to the debug message.
Would be good for debugging purposes.